hyrax/lib/tasks/create_missing_entities_and_deposit_works.rake

```diff
@@ -0,0 +1,21 @@
+namespace :works do
+  desc 'Create missing entities and deposit'
```

Comment:
I added `desc` although it works without it. If it doesn't exist, it won't be displayed in `rake -T`.